pallets/proposals/src/migration.rs

```diff
@@ -680,6 +681,77 @@ pub mod v6 {
     }
 }
 
+pub mod v7{
+    use super::*;
+
+    #[derive(Encode, Clone, Decode, Debug)]
+    pub struct MilestoneV7 {
+        pub project_key: u32,
+        pub milestone_key: u32,
+        pub percentage_to_unlock: u32,
+        pub is_approved: bool,
+        pub withdrawn: bool,
+    }
+    #[derive(Encode, Decode, Clone)]
+    pub struct ProjectV7<AccountId, Balance, BlockNumber> {
```

Comment:
This is meant to be the old Project type, the new project type is not defined here but defined in crate::Project
This should be the Project before you made the changes.